topostats/grains.py

```diff
@@ -656,12 +656,13 @@ def improve_grain_segmentation_unet(
         # https://github.com/keras-team/keras/issues/19441 which also has an experimental fix that we can try but
         # I haven't tested it yet.
 
-        # Important for when loading the model inievitably fails on users' machines.
-        LOGGER.info(f"Python executable: {os.sys.executable}")
-        LOGGER.info(f"Keras version: {keras.__version__}")
-        LOGGER.info(f"Model path: {unet_config['model_path']}")
-
-        unet_model = keras.models.load_model(unet_config["model_path"], compile=False)
+        try:
+            unet_model = keras.models.load_model(unet_config["model_path"], compile=False)
+        except Exception as e:
```

Comment:
This will likely have fallen foul of `pylint`'s complaint about broad exceptions.

Are there any examples of the types of exceptions that are raised that we could use?